src/ngraph/op/topk.hpp

```diff
@@ -161,8 +161,13 @@ namespace ngraph
                     copy_with_new_args(const NodeVector& new_args) const override;
 
                 virtual size_t get_version() const override { return 1; }
-                size_t get_axis() const { return m_axis; }
-                void set_axis(const size_t axis) { m_axis = axis; }
+                size_t get_axis() const { return m_normalized_axis; }
+                int64_t get_origin_axis() const { return m_axis; }
```

Comment:
Consider a different name.

Reply:
`get_specified_axis` ?

Reply:
Or turn it around - return `m_axis` from `get_axis()` and `m_normalized_axis` from `get_normalized_axis()`

Reply:
@tomdol's conventions seems to me the most clear.

Reply:
@arogowie-intel @tomdol I agree that `get_normalized_axis()` is the most clear, but current backends use `get_axis` method to read axis. I think that such change requires spec change.
My proposition is `get_raw_axis`